profiles/package.mask

```diff
@@ -17,6 +17,37 @@ dev-haskell/accelerate-io
 app-text/lit
 dev-haskell/highlighting-kate
 
+# Masked for removal. These require <dev-haskell/lens-4.19, but we're bumping
+# lens up to 4.19. All other upstreams have updated their dependency list to
+# support the new lens, but these have not. These also appear to be abandoned
+# projects.
+dev-haskell/reflex-basic-host
+dev-haskell/reflex-basic-socket
+dev-haskell/reflex-basic-wai
+
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2020-12-18)
+# Temporarily mask these, until ghc 8.10 is keyworded
+=dev-haskell/clash-prelude-1.2.5
```

Comment:
These probably need to be `>=` style depends, instead of `=` style depends to survive future bumps. Also dropping keywords in ebuilds would also match what we do for ghc and it's depends instead of masks.

Reply:
I don't have a problem changing this to `>=` from `=`, though actually by the time we're done reviewing this PR and it's ready for merge these should hopefully be gone, as @solpeth is almost done with `ghc 8.10`.

> dropping keywords in ebulids would also match what we do for ghc

Can you explain what you mean by this?

> it's depends instead of masks

So you mean the comment should read "Temporarily depends these, until..."?

Reply:
Instead of keeping `KEYWORDS="~amd64 ~x86"` in ebuilds that depend on `ghc-8.10` you can set `# keep in sync with ghc-8.10\n#KEYWORDS="~amd64 ~x86"` as we do for other ebuilds.

Reply:
Ah hah, yes this makes sense. So anything that is being masked due to ghc
compatibility, just remove the keywords rather than using package.mask, i
can do that!